packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/mask.tests.js

```diff
@@ -2409,3 +2411,17 @@ QUnit.module('ShadowDOM', {}, function() {
     });
 });
 
+QUnit.module('drag text', moduleConfig, () => {
+    QUnit.test('drag events should be prevented in a masked number box', function(assert) {
```

Comment:
```suggestion
    QUnit.test('drop event should be prevented in a masked number box (T1188135)', function(assert) {
```

Reply:
Fixed in [f0b0bf5f2a89e12ddc6f0d76ad534fe221a08aef].